### DISCUSSION TITLE
Some notes from the PS's [Ebrar]

### DISCUSSION THREAD

**Discussion by ebrarkiziloglu** [bounswe/bounswe2024group5#67] [General]
# PS Notes on March 4

* We are expected to have **"a few" (so 4-5 ? ) milestones** at the end of the semester.
* Such milestones should be arranged uniformly throughout the semester.
* The project consists of several milestones, which include several deliverables within.
* We could and are ENCOURAGED to use [ProjectLibre](https://www.projectlibre.com/) for keeping track of our project.
* We can use other softwares as well but we need to consult to teaching-staff first [also do not use any software that requires payment].

**Comment by ebrarkiziloglu** [comment]
# PS Notes on March 11
* We have a scenario of party planning company. They are functioning fully manually at the moment. Our job is to make it digital.
* We asked questions to the client (TAs) to learn about the requirements. 
* We wrote requirements pages [glossary, functional reqs. etc].
* We have **a perspective of the users** when writing the requirements.

**Reply by set120120** [reply]
- Do not specify any User Interfaces in requirements part, because it is in design part.
